hs_app_netCDF/models.py

```diff
@@ -51,18 +51,24 @@ def create(cls, **kwargs):
 
         :param kwargs: the 'value' in kwargs should be a dictionary
         """
-
+        value_arg_dict = None
         if 'value' in kwargs:
+            value_arg_dict = kwargs['value']
+        elif '_value' in kwargs:
+            value_arg_dict = json.loads(kwargs['_value'])
```

Comment:
I've already made the suggestion. If you want to deserialize it as JSON then you have to serialize it as JSON in the first place. converting a dictionary into a string is not JSON serialization. The string `_value` has to come from `json.dumps()` if you are going to give it to `json.loads()`